### DISCUSSION TITLE
Custom Themes

### DISCUSSION THREAD

**Discussion by rvndbalaji** [langfuse/langfuse#3273] [Ideas]
### Describe the feature or potential improvement

It would be amazing if we could expose the LangFuse colors and theme file so that we could customize it 
I would like it if I could have my company's branding and theme 

Since this isn't a mission critical request, a simple CSS file that can be edited on the UI would be perfect to start with

**Comment by marcklingen** [comment]
We recently added some ui customization capabilities as part of the Langfuse Enterprise Edition: https://langfuse.com/changelog/2024-08-14-ui-customization

We'd be open to add more functionality here. Please reach out if you are interested: enterprise@langfuse.com
